src/lambda_function/s3_event_config/template.yaml

```diff
@@ -45,6 +46,13 @@ Resources:
           S3_TO_GLUE_FUNCTION_ARN: !Ref S3ToGlueFunctionArn
           BUCKET_KEY_PREFIX: !Ref Namespace
 
+  LambdaInvokePermission:
+    Type: AWS::Lambda::Permission
+    Properties:
+      FunctionName: !GetAtt S3ToGlueFunction.Arn
+      Action: lambda:InvokeFunction
+      Principal: !Ref 'AWS::AccountId'
+
```

Comment:
This invoke permission is relatively broad, but [the docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-permission.html) don't mention being able to specify a specific IAM role. I couldn't find any additional information on this use case.

Reply:
It's also unclear to me if this is even necessary. The IAM role assumed by the github action has admin permissions and we might not need an additional `AWS::Lambda::Permission` here. I will do some testing.

Reply:
We don't need it.